src/main/java/de/tarent/entities/Course.java

```diff
@@ -71,6 +75,11 @@
             inverseJoinColumns = @JoinColumn(name = "category_id"))
     public List<Category> categoryList;
 
+    @JsonIgnore
```

Comment:
We have transient and JsonIgnore in this class. We definitely have to declare when to use which one.

Reply:
@Transient is for Persistence, @JsonIgnore is for JSON Serdes